package.json

```diff
@@ -195,6 +215,21 @@
         "command": "one.viewer.jsonTracer",
         "title": "json tracer",
         "category": "ONE"
+      },
+      {
+        "command": "one.viewer.metadata.show",
```

Comment:
Please add those commands to "commandPalette" and set "false".

For example, L399 of `package.json`,
![image](https://user-images.githubusercontent.com/17171963/193398116-df0a3cab-e0ff-4aab-abd2-e83b4ad96794.png)


This will conceal those commands from command palette.
Currently, they are shown as below. 
![image](https://user-images.githubusercontent.com/17171963/193398237-9c397939-cf52-4923-a690-0e86d279f689.png)

Reply:
All right, I'll apply that part.